src/main/java/com/GeorgesServer/app/Router.java

```diff
@@ -0,0 +1,28 @@
+package com.GeorgesServer.app;
+
+import com.GeorgesServer.app.com.GeorgesServer.handler.IHandler;
+
+public class Router {
+
+    private IHandler defaultHandler;
+    private IHandler formHandler;
+    private IHandler handler;
+
+    public Router(IHandler defaultHandler, IHandler formHandler) {
```

Comment:
every time you add a new handler you are going to have to change a bunch of things in many different places

Reply:
Alright then it's time to make a router config so I don't continue to change everything. That will also reduce the size of the PR's